debian8/guide/system/fs-part/installation-storage-partitioning/partition_for_srv.rule

```diff
@@ -20,4 +20,4 @@ severity: unknown
 references:
     anssi: NT28(R12)
 
-ocil: '<partition-check-macro part="/home " />'
+ocil: '{{{ partition_check(part="/home ") }}}'
```

Comment:
@matejak we should probably be consistent and either use single quotes or remove them.

Reply:
I agree.